packages/pixeloven/cli-addon-generators/src/templates/app-ssr/__mocks__/axios.ts

```diff
@@ -0,0 +1,75 @@
+import { AxiosRequestConfig } from "axios";
```

Comment:
We should cut these __mocks__ from the generator. This is starting to get into the specifics of app development. The generate should provide the bare min for creating an but not include any opinions on how the app itself might be used/created.